src/transformers/models/mask2former/image_processing_mask2former.py

```diff
@@ -733,9 +781,9 @@ def preprocess(
                 "torch.Tensor, tf.Tensor or jax.ndarray."
             )
 
-        if not is_batched(images):
-            images = [images]
-            segmentation_maps = [segmentation_maps] if segmentation_maps is not None else None
+        images = make_list_of_images(images)
```

Comment:
This is the additional fix to support fast_processor. Since there is a fast related testing script that gives `images` as a shape of [B,C,H,W]. However, `is_batched` function only considers the tuple, and list for the checking if it is batched, which makes this `images` to `List[torch.tensor[B,C,H,W]]`.
cc. @qubvel @yonigozlan